how-to/workspace-platform-starter/client/src/framework/platform/interopbroker.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from "@finos/fdc3";
 import type OpenFin from "@openfin/core";
 import type { AppIntent } from "@openfin/workspace-platform";
+import { getCenterPosition } from "workspace-platform-starter/utils-position";
```

Comment:
This include should be by relative reference, workspace-platform-starter imports should only be used by modules

Reply:
Yeah VSCode did that. Fixed.